pyresample/test/test_spherical.py

```diff
@@ -383,41 +429,41 @@ def test_is_inside(self):
 
         polygon2 = SphPolygon(np.deg2rad(vertices))
 
-        self.assertTrue(polygon1._is_inside(polygon2))
+        assert (polygon1._is_inside(polygon2))
```

Comment:
These parentheses aren't necessary. Could you please remove them on this line and all other assert lines?